.github/workflows/code_checks.yml

```diff
@@ -66,3 +33,20 @@ jobs:
                 ':!**/*.sln' \
                 ':!**/*.vcproj' \
                 ':!**/*.xpm'
+
+
+  check-mixed-eol:
```

Comment:
I don't know if it's worth separating different checks in different jobs or combine them all in a single job and just use different steps (and allow the job to run to completion even if one of the steps fails).

It looks like the latter would be a more efficient use of GitHub resources...

Reply:
Setting up and checking out seems to take only a few seconds. And I've been struggling with exit codes too much to feel motivated to do it right now.